Sources/KVKCalendar/CalendarData.swift

```diff
@@ -19,14 +19,37 @@ struct CalendarData {
     var yearsCount = [Int]()
     
     init(date: Date, years: Int, style: Style) {
-        self.date = date
-        self.style = style
-        
         // count years for calendar
         let indexesYear = [Int](repeating: 0, count: years).split(half: years / 2)
+        
+        self.init(date: date, style: style, indexesYear: indexesYear)
+    }
+
+    init(date: Date, style: Style, startYear: Int, endYear: Int) {
```

Comment:
Because only the year was supported in the past, I did not design these two parameters as `Date` type.

If they were designed as `Date` type, users might misunderstand that the start and end ranges would be accurate to the date, but in fact there are only years.